images/patroni/tests/main.tf

```diff
@@ -0,0 +1,14 @@
+terraform {
+  required_providers {
+    oci = { source = "chainguard-dev/oci" }
+  }
+}
+
+variable "digest" {
+  description = "The image digest to run tests over."
+}
+
+data "oci_exec_test" "version" {
+  digest = var.digest
+  script = "docker run --rm $IMAGE_NAME version"
+}
```

Comment:
It looks like Patroni has some tests in the repo: https://github.com/zalando/patroni/tree/master/kubernetes

Could you try using an `imagetest_harness_k3s` test and apply one of those examples using our image? That will ensure that our image runs in a K8s environment.

Reply:
Grabbing this https://github.com/zalando/patroni/blob/master/kubernetes/patroni_k8s.yaml#L36 and editing the `image` to use ours will be a good test I think.